lua/core/compe.lua

```diff
@@ -82,6 +82,14 @@ M.setup = function()
     end
   end
 
+  local is_emmet_active = function()
+    local utils = require "lsp.utils"
+    if utils.is_client_active "emmet_ls" then
```

Comment:
As discussed, please check if emmet LSP client is attached to the current buffer/filetype.